00-RELEASENOTES

```diff
@@ -11,6 +11,17 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 7.2.5 GA  -  Released Tue 16 Apr 2024
+================================================================================
+
+Upgrade urgency LOW: This is the first stable release of Valkey 7.2.5.
+
+Changes to support Valkey branding
+=========
+* Update template config files to remove references of redis.io and replace them
+  with valkey.io. (#320)
```

Comment:
nit. remove "." at the end? this is not a complete sentence? 
```suggestion
  with valkey.io (#320)
```

Reply:
Is it not a complete sentence? If "Update" is imperative, the sentence doesn't need a subject. (I might be wrong.)

Reply:
ah you are right 
now I have questions for other imperative sentences :)